utils/jwt-helpers.js

```diff
@@ -0,0 +1,12 @@
+import jwt from 'jsonwebtoken';
+
+
+const jwtTokens = ({ email, password }) => {
```

Comment:
How do you ensure that this will work? It might be a good idea to introduce a try/catch block here.